db/schema.rb

```diff
@@ -52,6 +52,16 @@
     t.integer "posts_counter", default: 0
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.text "bio"
+    t.string "confirmation_token"
+    t.datetime "confirmed_at"
+    t.datetime "confirmation_sent_at"
+    t.string "unconfirmed_email"
+    t.string "encrypted_password"
+    t.datetime "remember_created_at"
+    t.string "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.index ["confirmation_token"], name: "index_users_on_confirmation_token", unique: true
```

Comment:
- [ ] I think you have misunderstood the devise setup. These columns including 'email' will be added by the migration named 'add_devise_to_users' which we added through this command 'rails g devise user'. But it seems like you have removed the necessary lines from that file and added these columns yourself. Please remove these migration files first and then rerun 'rails g devise user' and uncomment the 'confirmable' section. You need to remove 'email' column first from users to not face an error. Kindly implement this.

Reply:
Thank you very much sir for the review and suggestions, the suggested changes are kindly implemented